### PR TITLE
Add location property to objects

### PR DESCRIPTION
**Pull request by davetsay**
## Overview
Addresses #24 

## Testing Instructions
1. Observe dictionary items and see whether they are marked as aliases or not

## Author Checklist
| | |
| --- | :---: |
| Changes address original issue? | Y |
| Unit tests included and/or updated with changes? | N/A |
| Command line build passes? | Y |
| Changes have been smoke-tested? | Y |
| Testing instructions included? | Y |

### CODE REVIEW

**Comment by davetsay** [comment]
@jvigliotta , only assigning to @akhenry for review per new reviewer guidelines but please also have a look if you have time.

**Review by jvigliotta — COMMENTED** [review]
LGTM!

**Review by jvigliotta — APPROVED** [review]
## Reviewer Checklist
Changes appear to address issue? Yes
Appropriate unit tests included? N/A
Code style and in-line documentation are appropriate? Yes
Commit messages meet standards? Yes
Has associated issue been labelled unverified? (only applicable if this PR closes the issue) Yes

File: src/providers/object-provider.js [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import {
     qualifiedNameToId,
-    accumulateResults
+    accumulateResults,
+    makeKeyString
```
Comment (akhenry) [inline src/providers/object-provider.js:4]: The Open MCT object api exposes this function, so we could avoid duplicating this function in this repo by just passing the open mct instance into the constructor for `YamcsObjectProvider`

**Review by akhenry — CHANGES_REQUESTED** [review]
This looks really great, thanks @davetsay! Just one minor suggestion - I think we can avoid duplicating the `makeKeyString` function. See comment.

File: src/providers/object-provider.js (hunk 1 above)
Comment (jvigliotta) [reply src/providers/object-provider.js:4]: Dang! I thought so too, shoulda said something : )

Comment (davetsay) [reply src/providers/object-provider.js]: done. I had thought so too.... and then didn't : (

**Review by akhenry — APPROVED** [review]
Looks great, thanks

**Pull request merged**